src/roles/vnsutil-predeploy/tasks/openstack.yml

```diff
@@ -0,0 +1,87 @@
+---
+# TODO:
+# Check for existing stack or vms
+
+- name: Display if skipping VNSUtil  predeploy
+  debug:
+    msg:
+      - "******************************************************"
+      - "Skipping VNSUTIL predeploy because it is already present"
+      - "******************************************************"
+  when: vnsutil_present
+
+- block:
+
+  - name: Set local variable with upgrade status
```

Comment:
We don't support upgrade for VNS utils. These tasks should be removed.

Reply:
Done!